bears/yml/RAMLLintBear.py

```diff
@@ -0,0 +1,19 @@
+from coalib.bearlib.abstractions.Linter import linter
+
+
+@linter(executable='ramllint',
+        use_stdout=False,
+        use_stderr=True,
+        output_format='regex',
+        output_regex=r'(?P<severity>error) (?P<filename>.+)\n  (?P<message>.+)')
```

Comment:
According to https://github.com/QuickenLoans/ramllint/blob/master/src/cli.js#L15 there are probably three logging levels. Modify regex accordingly.